crates/evm-prover/src/prover/prover.rs

```diff
@@ -141,6 +103,22 @@ impl AppContext {
     }
 }
 
+// ProofJob is a basic struct to the track proof generation per block event.
+struct ProofJob {
+    height: u64,
+    blobs: Vec<Blob>,
+}
+
+impl ProofJob {
+    fn new(height: u64, blobs: Vec<Blob>) -> Self {
+        Self { height, blobs }
+    }
+}
+
+// TODO: Add these as fields to the BlockExecProver to make configurable?
+const QUEUE_CAP: usize = 256;
+const CONCURRENCY: usize = 16;
```

Comment:
We should add these settings to config to allow them to be tweaked if needs be. We can create an issue for this and do in a follow up.